### ISSUE TITLE
Train loss during training not changing.

### ISSUE THREAD

**Issue by NikitaRA** [facebookresearch/dinov2#298]
I started train with command
python3 -m dinov2.run.train.train --config-file dinov2/configs/train/vitl16_short.yaml

with yaml params:

train:
  dataset_path: ImageNet:split=TRAIN
  batch_size_per_gpu: 32
  OFFICIAL_EPOCH_LENGTH: 2500
student:
  arch: vit_base
  patch_size: 14
teacher:
  momentum_teacher: 0.994
optim:
  weight_decay_end: 0.2
  base_lr: 2.0e-04  
  layerwise_decay: 1.0
  epochs: 100
  warmup_epochs: 1

I create my own dataset with imageNet-like structure, but during ~50 epochs steps - loss changing in interval 0.01% from 1st epoch, and still flat.
I start train without any specific pretrained weights, is it correct?
What im doing wrong?

**Comment by qasfb** [comment]
It looks like you are training in a non-distributed setup, with a total batch size of 32. We do not have much experience with this setting, as we use a total batch size of 2048 in our distributed setups; my gut feeling tells me it would not work well.

**Comment by qasfb** [comment]
Probably this is the issue. I have never trained DINOv2 setups with small batch sizes such as 32; we use a batch size of 2048 in our work, and the optimization problem is very different in that case. Unfortunately I don't have any solution to offer to you, and at this point I can not even confirm there is a way to make it work with a batch size of 32.

**Comment by NikitaRA** [comment]
> Are you running a distributed setup ?

No, just 1 A100 40 Gb

**Comment by NikitaRA** [comment]
> Probably this is the issue. I have never trained DINOv2 setups with small batch sizes such as 32; we use a batch size of 2048 in our work, and the optimization problem is very different in that case. Unfortunately I don't have any solution to offer to you, and at this point I can not even confirm there is a way to make it work with a batch size of 32.

Well, that ain't good. Thanks anyway.
